saleor/graphql/order/tests/mutations/test_draft_order_complete.py

```diff
@@ -1158,9 +1160,11 @@ def test_draft_order_complete_force_tax_calculation_when_tax_error_was_saved(
 
 @freeze_time()
 @override_settings(PLUGINS=["saleor.plugins.webhook.plugin.WebhookPlugin"])
+@patch("saleor.order.calculations.validate_tax_data")
```

Comment:
Same here

Reply:
I mock the function to skip the validation, since the test use dummy, invalid tax data